src/base/LMIcon/types.tsx

```diff
@@ -0,0 +1,13 @@
+import { ImageStyle, ViewStyle } from "react-native";
+
+export interface LMIconProps {
+    type: 'png',
+    assetPath?: string,
+    color?: string,
+    size?: number,
+    iconStyle?: ImageStyle,
+    boxFit?: 'center' | 'contain' | 'cover' | 'repeat' | 'stretch',
+    boxStyle?: ViewStyle
+}
```

Comment:
add comments for each props, what is represents

Reply:
resolved